ci-operator/step-registry/baremetal/lab/pre/dhcp-conf/baremetal-lab-pre-dhcp-conf-commands.sh

```diff
@@ -62,4 +62,5 @@ timeout -s 9 10m ssh "${SSHOPTS[@]}" "root@${AUX_HOST}" bash -s -- \
   "'${DHCP_CONF}'" "'${DHCP_CONF_OPTS}'" "'${CLUSTER_NAME}'" <<'EOF'
   echo -e "${1}" > /opt/dnsmasq/hosts/hostsdir/"${3}"
   echo -e "${2}" > /opt/dnsmasq/hosts/optsdir/"${3}"
+  echo -e "dhcp-option=option6:router,[fd99:2222:3456::1]" >> /opt/dnsmasq/hosts/optsdir/"${3}"
```

Comment:
```suggestion
```

Reply:
we can't merge this. so remember to remove before we merge and have a test once this is available in the bastion host.

Reply:
Sure, I will keep in mind.